base/deprecated.jl

```diff
@@ -2109,6 +2109,12 @@ end
     @deprecate chol!(x::Number, uplo) chol(x) false
 end
 
+# deprecate RowVector{T}(shape...) constructors to RowVector{T}(uninitialized, shape...) equivalents
+@deprecate RowVector{T}(n::Int) where {T}               RowVector{T}(uninitialized, n)
+@deprecate RowVector{T}(n1::Int, n2::Int) where {T}     RowVector{T}(uninitialized, n1, n2)
+@deprecate RowVector{T}(n::Tuple{Int}) where {T}        RowVector{T}(uninitializd, n)
```

Comment:
`uninitializd` -> `uninitialized`
I would have thought you were a master of spelling uninitialized after this series of PR's :wink:

Reply:
I consistently make either of two mistakes: `unintialized`, or `uninitializd`. Seems I have trouble typing vowels? 😄

Reply:
Fixed. Thanks! :)